base/logger.go

```diff
@@ -34,3 +38,45 @@ func logCollationWorker(collateBuffer chan string, logger *log.Logger, bufferSiz
 		}
 	}
 }
+
+func runLogDeletion(logDirectory string, logLevel string, maxSize int) {
```

Comment:
No indication of what `maxSize` means here.

```suggestion
func runLogDeletion(logDirectory string, logLevel string, sizeLimitMB int) {
```